client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -273,7 +279,12 @@ const PersonalPurchase = ( {
 					</ButtonComponent>
 
 					{ isNewPurchaseFlowEnabled && (
-						<ButtonComponent variant="secondary" onClick={ handleCheckoutPostponed }>
+						<ButtonComponent
+							variant="secondary"
+							isBusy={ isWPCOMSite ? undefined : isPosponeBusy } // for <Button />
```

Comment:
might be easier to understand to just pass  `isPosponeBusy` without `isWPCOMSite` if there's no side effect? Same for the following line.

Reply:
I had that initially, but I was getting React console errors that I was passing `false` to `busy` when that component didn't expect that prop.